src/render/SDL_render.c

```diff
@@ -553,6 +563,35 @@ QueueCmdCopyEx(SDL_Renderer *renderer, SDL_Texture * texture,
     return retval;
 }
 
+#if SDL_HAVE_RENDER_GEOMETRY
+static int
+QueueCmdGeometry(SDL_Renderer *renderer, SDL_Texture *texture,
+        const float *xy, int xy_stride,
+        const int *color, int color_stride,
+        const float *uv, int uv_stride,
+        int num_vertices,
+        const void *indices, int num_indices, int size_indice,
```

Comment:
Singular form is **index**, not **indice**.

Reply:
I have changed 'size_indice', to 'size_indices'. Is that right?
maybe all should be index, and also vertex ?

Reply:
I'd use something like a "index_size", because we're talking about the size of a single array element, not size of an entire array. But, I guess, it is a matter of taste.